### DISCUSSION TITLE
Feature Request: Provide project "Move all in Column" for moving issues to a new project

### DISCUSSION THREAD

**Discussion by Colengms** [community/community#6572] [Projects and Issues]
We use projects for release iterations.  Often we are unable to complete all of the tasks/issues before shipping, and need to move those issues individually to a new (or existing) project.  It would be great if GitHub could better facilitate this workflow by making it easy to move whatever remains in the project (usually just a single "TODO" column) to the next iteration.

**Comment by ben-anderson** [comment]
I'd love to have this functionality as well. Having to move a bunch of issues to a new project is very tedious.
